# GetBlocksPoses.srv

---
bool success
string message
vision_planner/BlockPose[] block_poses
